### DISCUSSION TITLE
希望自己实现邮件转发功能

### DISCUSSION THREAD

**Discussion by cccqcccq** [Jinnrry/PMail#196] [Q&A]
目前自己准备打算做一个网站，希望能够自己实现一个邮件转发的功能，尝试阅读了一下源码但是感觉一窍不通，可以指点一下代码中哪部分可以获取到收取到的邮件内容吗。
目前猜测是stmp_server包中的saveEmail代码，是否可以在里面写入发送邮件的代码
![image](https://github.com/user-attachments/assets/5f565249-ccf0-447b-9fb8-fec16509049a)

**Comment by Jinnrry** [comment]
https://github.com/Jinnrry/PMail/wiki/%E6%8F%92%E4%BB%B6%E5%BC%80%E5%8F%91%E8%AF%B4%E6%98%8E

不需要你改源码，你写一个你自己的插件就行

**Comment by Jinnrry** [comment]
此外，设置里面有收信规则。配置收信规则也支持邮件转发

**Reply by cccqcccq** [reply]
好，感谢大佬
